### PR TITLE
Rework of LB.

### PR DESCRIPTION
**Pull request by Nanu308**
:cl: Nanu308
maptweak: REWORK OF LIGHT BRIGADE.
maptweak: Moved Nuke, AI, Navcom.
maptweak: Made a new way from Briefing to ForePortside Hangar.
maptweak: Added some of that Flare
maptweak: Added better new Engines that do not need cooling
maptweak: Added an Atmos system that functions.
maptweak: Changed Medbay, All should be there. If you miss anything say so.
maptweak: Removed old atmos system
/:cl:

### CODE REVIEW

**Comment by BDpuffy420** [comment]
Travis mad

**Comment by Nanu308** [comment]
### **I WILL KILL THIS BOT**

**Comment by BDpuffy420** [comment]
PR still failing due to other reason not related to deso's GM

File: maps/UNSC_Halberd_Class/areas.dm [hunk 1]
```diff
@@ -1,68 +1,125 @@
...
-
-/area/destroyer/unsclightbrigade/storage
-	name = "UNSC Light Brigade Storage Room"
-
-/area/destroyer/unsclightbrigade/crewbathroom
-	name = "UNSC Light Brigade Crew Bathroom"
-
-/area/destroyer/unsclightbrigade/centralhall
-	name = "UNSC Light Brigade Central Hallway"
+	/area/destroyer/unsclightbrigade
```
Comment (caelaislinn) [inline maps/UNSC_Halberd_Class/areas.dm]: This is failing travis, there shouldn't be an indent here

**Review by caelaislinn — CHANGES_REQUESTED** [review]
Fix that single indent (remove it so it aligns with the edge of the file) and it will fix all the travis failures

File: maps/UNSC_Halberd_Class/areas.dm (hunk 1 above)
Comment (Nanu308) [reply maps/UNSC_Halberd_Class/areas.dm]: I believe this have been fixed now

**Comment by X0-11** [comment]
set to DNM, awaiting further update to changelog

**Comment by Nanu308** [comment]
I will make a full changelogg tomorrow/today depending if all issues get removed

**Review by caelaislinn — APPROVED** [review]

**Comment by caelaislinn** [comment]
Travis failure is achlys map

**Pull request merged**